src/index.js

```diff
@@ -0,0 +1,132 @@
+import './style.css';
+import Draggables from './drag';
+import statusCheck from './status';
+import uiUpdate from './updates';
+
+class TaskList {
+  refereceList = [];
+
+  container = document.querySelector('.container');
```

Comment:
[Optional] I think separating logic from DOM manipulation is a best practice. It will help you keep your code cleaner and you'll know that everything logic-related will be in one place.

I'll mark this as Optional because it requires a lot of time to refactor your logic, but if you ever find the time give it a try or just keep it in mind for your next project.